OpenRA.Mods.Common/Activities/Attack.cs

```diff
@@ -116,7 +118,8 @@ protected virtual Activity InnerTick(Actor self, AttackBase attack)
 
 			var targetedPosition = attack.Info.AttackTargetCenter ? Target.CenterPosition : Target.Positions.PositionClosestTo(pos);
 			var desiredFacing = (targetedPosition - pos).Yaw.Facing;
-			if (facing.Facing != desiredFacing)
+			if ((facingTolerance <= 1 && facing.Facing != desiredFacing) ||
+				Math.Abs((facing.Facing - desiredFacing) % 256) > facingTolerance)
```

Comment:
Yes, wrap it into range using NormalizeFacing and then compare against facingTolerance and 255-facingTolerance.  That is what our other code that cares about facing offsets does.